runtime/browser/xwalk_runner.cc

```diff
@@ -72,6 +72,10 @@ void XWalkRunner::PreMainMessageLoopRun() {
     extension_service_.reset(new extensions::XWalkExtensionService(
         app_extension_bridge_.get()));
 
+#if !defined(OS_ANDROID)
+  if (cmd_line->HasSwitch(switches::kXWalkDisableSaveFormData))
```

Comment:
put it to "XWalkBrowserMainParts::PreMainMessageLoopRun" and "#if !defined(OS_ANDROID)" won't be needed